uber/templates/registration/attendee_data.html

```diff
@@ -0,0 +1,125 @@
+{% set admin_area=True %}
+{% set clientside_bool=True %}
+{% import "fields/attendee.html" as attendee_fields with context %}
+<script src="//cdnjs.cloudflare.com/ajax/libs/validate.js/0.13.1/validate.min.js"></script>
+<script type="text/javascript">
+createDateTextEntries();
+{% if attendee.is_new %}
+$.field('placeholder').prop('checked', true);
+{% endif %}
+var saveAndClose = function() {
+  $("#attendee_form").submit();
+  $("#attendee_modal").modal('hide');
+};
+  
+var updateAttendee = function(event){
+    event.preventDefault();
+    toastr.clear();
+
+    var formData = $("#attendee_form").serializeArray();
+    var old_hash = window.location.hash;
+    $.each(validate.collectFormValues($("#attendee_form"), {nullify:false}), function(k, v) {
+        if (v === false || v === null) {
+        formData.push({name: k, value: (v === null) ? "" : v});
+        } else if (v === true) {
+        formData.find(input => input.name == k).value = v;
+        }
+    });
+    $.ajax({
+        method: 'POST',
+        url: '../registration/update_attendee',
+        data: formData,
+        success: function (json) {
+            if (json.success) {
+            toastr.info(json.message);
```

Comment:
Although it's "only" a style issue, you should probably indent the code inside the if-statement, since it confused me at first glance as to whether the lines below were all part of this if-block.

Reply:
Oops, good catch!